kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -290,6 +290,11 @@ form .input-group {
     background-color: #40775F !important;
 }
 
+.login-btn, btn-block {
+    display: block;
+    width: 100%;
```

Comment:
Check the latest commit:

a) Login dialog loaded
![simplified-login-dialog2](https://cloud.githubusercontent.com/assets/1457929/6995799/5aa735de-db5f-11e4-96f0-0cbdf43cf827.png)

b) Wrong username no password
![simplified-login-dialog3](https://cloud.githubusercontent.com/assets/1457929/6995802/7a0e1550-db5f-11e4-8f35-28d1b620654b.png)

c) Wrong username & password
![simplified-login-dialog4](https://cloud.githubusercontent.com/assets/1457929/6995806/89fc02ec-db5f-11e4-9b81-fb9d9a3be357.png)

One thing I noticed is that the error message tooltip stays on username even when it is correctly inputed for the given facility, and the password is the incorrect one. I'm guessing lines 219-220 in /facility/forms.py?